client/src/connection/itc/index.ts

```diff
@@ -368,6 +369,10 @@ export class ITCSession extends Session {
       return l10n.t("This platform does not support this connection type.");
     }
 
+    if (/Init error/.test(msg)) {
```

Comment:
nit: since it's specific to loading libraries, how about a more specific name? something like
```suggestion
    if (/LoadingInterop error/.test(msg)) {
```

Reply:
Addressed in https://github.com/sassoftware/vscode-sas-extension/pull/1409/commits/a2b4f0e8bd572d3067f6a9b2c1023d45aa88fde4